test/pouch_cli_ps_test.go

```diff
@@ -1,57 +1,100 @@
 package main
 
 import (
-	"os/exec"
+	"strings"
 
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
 	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
 // PouchPsSuite is the test suite fo help CLI.
-type PouchPsSuite struct {
-}
+type PouchPsSuite struct{}
 
 func init() {
 	check.Suite(&PouchPsSuite{})
 }
 
 // SetUpSuite does common setup in the beginning of each test suite.
 func (suite *PouchPsSuite) SetUpSuite(c *check.C) {
-	SkipIfFalse(c, IsLinux)
-
-	// Pull test image
-	cmd := exec.Command("pouch", "pull", testImage)
-	cmd.Run()
-}
+	SkipIfFalse(c, environment.IsLinux)
 
-// SetUpTest does common setup in the beginning of each test.
-func (suite *PouchPsSuite) SetUpTest(c *check.C) {
-	// TODO
-}
-
-// TearDownSuite does cleanup work in the end of each test suite.
-func (suite *PouchPsSuite) TearDownSuite(c *check.C) {
-	// TODO: Remove test image
+	c.Assert(environment.PruneAllContainers(apiClient), check.IsNil)
+	command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
 }
 
 // TearDownTest does cleanup work in the end of each test.
 func (suite *PouchPsSuite) TearDownTest(c *check.C) {
-	// TODO add cleanup work
+	environment.PruneAllContainers(apiClient)
 }
 
 // TestPsWorks tests "pouch ps" work.
+//
+// TODO: check more value, like id/runtime.
 func (suite *PouchPsSuite) TestPsWorks(c *check.C) {
+	name := "ps-normal"
+
+	// create
+	{
+		command.PouchRun("create", "--name", name, busyboxImage).Assert(c, icmd.Success)
+
+		res := command.PouchRun("ps").Assert(c, icmd.Success)
+		kv := psToKV(res.Combined())
+
+		c.Assert(kv[name].status, check.Equals, "created")
+		c.Assert(kv[name].image, check.Equals, busyboxImage)
+	}
+
+	// running
+	{
+		command.PouchRun("start", name).Assert(c, icmd.Success)
```

Comment:
We need to use `pouch run` to replace `create` and `start`, if issue https://github.com/alibaba/pouch/issues/351 is solved.

Reply:
I think there is no need to use `run` command, because I want to check the `created`, `running` and `stopped`. If I use `run`, I will miss the `created` status. :)

Reply:
make sense